docs/workshop/model_development.md

```diff
@@ -46,22 +46,44 @@ For an example `.env` file refer to this [sample env file](../../notebooks/time-
 For faster download, choose a repository which does not have too many PRs. For example purposes, we have chosen a repository with ~1000 PRs
```

Comment:
Can we move Git cloning to a separate section like


## Clone the Repository

This is an important step to get right for successfully being able to run the jupyter notebooks.

* Go to [Jupyterhub](https://jupyterhub-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/hub/spawn) and select the `Openshift CI Analysis Notebook Image` and the “Workshop” container size.

If you do not have access to the Jupyterlab instance being used for this workshop, move to chapter on [Onboarding](./onboarding.md).

* Open the terminal and clone your forked Github repository in Jupyterhub.

First set up your github configuration in Jupyterhub

```
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
```

Optional: You can also [generate an SSH key](https://docs.github.com/en/authentication/managing-commit-signature-verification) and add it to your github account at https://github.com/settings/keys.

`git clone https://github.com/{YOUR-USERNAME}/ocp-ci-analysis`

If you have not forked the repository, move to chapter on [Set up Git environment](./git_setup.md).